LSP.sublime-settings

```diff
@@ -111,8 +111,12 @@
   // Show symbol action links in hover popup if available
   "show_symbol_action_links": false,
 
-  // Disable Sublime Text's explicit and word completion.
-  "only_show_lsp_completions": false,
+  // Disable Sublime Text's snippet completions.
+  "inhibit_explicit_completions": false,
+
+  // Disable Sublime Text's word completions. When set to `true`, this also disables Sublime Text's internal completion
+  // sorting algorithm and instead uses the sorting defined by the relevant language server.
+  "inhibit_word_completions": true,
```

Comment:
Have you thought of having one setting that takes an array of strings instead? I think it might be easier to manager for us and for users.

Reply:
Hmm, I don't think there will be other flags in the foreseeable future.

Reply:
Sorry for the late reply. 
I would also agree with @rchl that a list would me more suitable. 

And I would also like to point out that we should be careful how we name settings. 
It is easy to forget that not all ST are power users, so we should be make the settings really simple to understand.

```
// Exclude from the auto complete:
// "snippets", "words"
"exclude_completions": []
```
we can also have a similar setting for the quick_panel symbol list(ctrl+r) to

```
// exclude symbols from the symbol list:
// "property", "class", "method", ... the rest of SymbolKind but only lowercase
"exclude_symbols": ["property"]
```